cassandane/Cassandane/Cassini.pm

```diff
@@ -131,11 +131,11 @@ sub new
 
 sub instance
 {
-    my ($class) = @_;
+    my ($class, %params) = @_;
 
     if (!defined $instance)
     {
-        $instance = Cassandane::Cassini->new();
+        $instance = Cassandane::Cassini->new(%params);
```

Comment:
This is kind of awkward, because if you call `Cassandane::Cassini->instance(%myparams)` you would reasonably expect to get back a Cassini object initialised with those params, but the actual result would depend on whether or not something else already initialised the singleton, which smells like an annoying race condition waiting to happen